docs/source/install.rst

```diff
@@ -160,6 +160,15 @@ Or if you wish to contribute to LiberTEM, follow these steps instead :
 
 For more information about `forking a repository
 <https://help.github.com/en/github/getting-started-with-github/fork-a-repo>`_.
+If you have limited experience using git, consider going through following
```

Comment:
Maybe "For a beginner-friendly introduction to git and GitHub, ..."? That's more from the perspective of what new users might want.

Also "going through the following", note the "the". :-)